src/api/app/assets/stylesheets/webui/notifications_redesign/notifications.scss

```diff
@@ -7,19 +7,29 @@ body.notifications-redesign  {
     grid-template-columns: 1fr auto;
     grid-template-rows: auto;
     grid-template-areas:
-      "notifiable notifiable"
-      "content content"
-      "who-when action";
+      "notifiable actions"
+      "content content";
     column-gap: 0.5rem;
     row-gap: 0.25rem;
-    align-items: start;
 
     .notifiable { grid-area: notifiable; }
-    .who-when { grid-area: who-when; }
-    .action { grid-area: action; }
+    .actions { grid-area: actions; }
     .content { grid-area: content; }
   }
 
+  .notifications-who-where-grid-container {
```

Comment:
Is there a reason you don't use the bootstrap grid? 

Why don't you stack the avatars and the test on mobile?

Reply:
@hennevogel, I was able to get rid of the custom grids.
The avatar's area and the text area are displayed in two different rows when their content is too large.